### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by larajanecka**
Adding eslint

### CODE REVIEW

File: src/js/pages/Events.jsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export default class Events extends React.Component {
   }
 
 	getEvents() {
+    console.log(EventStore.events);
```
Comment (StevenLocke) [inline src/js/pages/Events.jsx]: Extra print statement.

File: src/js/pages/Home.js [hunk 2]
```diff
@@ -1,79 +0,0 @@
-import React from "react";
```
Comment (StevenLocke) [inline src/js/pages/Home.js]: This file should remain.

File: src/js/pages/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import React from "react";
```
Comment (StevenLocke) [inline src/js/pages/Home.jsx]: This is out of date. Should replace with the old js file.

**Review by StevenLocke — CHANGES_REQUESTED** [review]
I think, aside from Home.js being replaced with an old version, this is good.

**Review by TheKevJames — APPROVED** [review]
LGTM but @StevenLocke should approve this too.

File: package.json [hunk 4]
```diff
@@ -17,6 +17,7 @@
     "extract-text-webpack-plugin": "^1.0.1",
     "flux": "^2.1.1",
     "history": "^1.17.0",
+    "immutable": "^3.8.1",
```
Comment (StevenLocke) [inline package.json:20]: I don't think this is still necessary.

**Review by StevenLocke — APPROVED** [review]

**Pull request merged**